tests/checkpatch_wrapper_test.sh

```diff
@@ -42,31 +59,80 @@ function oneTimeTearDown
   rm -rf "$FAKE_KERNEL"
 }
 
-function checkpatch
-{
-  res=$(execute_checkpatch "$FAKE_KERNEL/samples/codestyle_$1.c" 2>&1)
-  [[ "$res" == *"${!MSG[$1]}" ]]
-  assertTrue "Checkpatch should output: ${!MSG[$1]}" $?
-}
-
-function testWarning
+function warning_Test
 {
   checkpatch "warning"
 }
 
-function testError
+function error_Test
 {
   checkpatch "error"
 }
 
-function testChecks
+function checks_Test
 {
   checkpatch "check"
 }
 
-function testCorrect
+function correct_Test
 {
   checkpatch "correct"
 }
 
+function invalid_path_Test
+{
+  local build_fake_path
+  local output
+  local ret
+
+  build_fake_path="$RANDOM/$RANDOM/$RANDOM/xptolala"
+
+  output=$(execute_checkpatch "$build_fake_path")
+  ret="$?"
+  if [[ "$ret" != 22 ]]; then
+    assertFalse "We forced an invalid path and we expect an error" "$?"
+  fi
+}
```

Comment:
Hmm, it seems a little odd to use `assertFalse` inside the if statement, since the condition was already checked. You could replace `assertFalse` simply by `fail <message>` since the code will only get inside it when the test fails.

However, I would prefer to refactor this to:
```shell
execute_checkpatch "$build_fake_path"
ret = $?
assertTrue "We forced an invalid path and we expect an error" '[[ $ret == 22 ]]'
```

(I.e. to use `assertTrue`, giving the condition string. Also removed `output` since it's not used and removed the quotes from `$ret` and `$?` since they will always be a single number)